app/models/request.rb

```diff
@@ -563,6 +563,11 @@ def product_line
   def manifest_processed!
   end
 
+  def self.get_all_comments(request)
```

Comment:
A minor thing, but should this be a class method or an instance method? It's defined on `request` model, and takes in a `request` (very much like `self` in Python and Rust, `this` in Java, TypeScript, and C++) so I would imagine it operates on the given instance, making it a candidate for an instance method?

Reply:
> A minor thing, but should this be a class method or an instance method? It's defined on `request` model, and takes in a `request` (very much like `self` in Python and Rust, `this` in Java, TypeScript, and C++) so I would imagine it operates on the given instance, making it a candidate for an instance method?

if it is an instance method, should it live in the controller?

Reply:
It looks like it should be in the `request` model as it takes in the current request (current instance). Anyway, I will approve this since it accomplishes what you're trying to do.